server/server.go

```diff
@@ -252,6 +254,16 @@ func NewServer(logger *zap.Logger, version string, host string, port int, fronte
 	return s
 }
 
+// SetLicense will get and parse valid license and set into application.
+func (server *Server) SetLicense(licenseService enterprise.LicenseService) {
+	license, err := licenseService.ParseLicense()
+	if err != nil {
+		server.l.Warn("Cannot found valid license", zap.String("error", err.Error()))
+	}
+
```

Comment:
if we successfully load the license, let's print out to indicate the license info

Reply:
Updated, print info in `parseClaims` func